infrastructure/stacks/development-pipeline/variables.tf

```diff
@@ -0,0 +1,154 @@
+##########################
+# INFRASTRUCTURE COMPONENT
+##########################
+
+############
+# AWS COMMON
+############
+
+variable "aws_profile" {
+  description = "The AWS profile"
+}
+
+variable "aws_region" {
+  description = "The AWS region"
+}
+
+variable "aws_account_id" {
+  description = "AWS account Number for Athena log location"
+}
+
+variable "aws_account_id_nonprod" {
+  description = ""
+}
+variable "aws_account_id_mgmt" {
+  description = ""
+}
+variable "aws_account_id_prod" {
+  description = ""
+}
+
+variable "aws_account_id_identities" {
+  description = ""
+}
+
+variable "aws_account_id_tools" {
+  description = ""
+  default     = "00000000000"
+}
+variable "aws_account_id_live_parent" {
+  description = ""
+}
+# ##############
+# # TEXAS COMMON
+# ##############
+
+
+variable "code_pipeline_branch_name" {
+  description = ""
+}
+
+
+variable "github_owner" {
+  description = ""
+}
+
+variable "github_token" {
+  description = ""
+}
+
+variable "AWS_ACCOUNT_ID_LIVE_PARENT" {
```

Comment:
Can the uppercase  `AWS_ACCOUNT` variables be removed and instead use the lowercase versions `aws_account`. As the lowercase variables are automatically exported so don't have to be manually exported.